test/extended/networking/networkpolicy.go

```diff
@@ -330,6 +330,65 @@ var _ = Describe("[Area:Networking] NetworkPolicy", func() {
 			testCannotConnect(f, nsA, "client-a", service, 80)
 			testCanConnect(f, nsB, "client-b", service, 80)
 		})
+
+		It("should enforce policy based on NamespaceSelector and PodSelector [Feature:OSNetworkPolicy]", func() {
+			nsA := f.Namespace
+			nsBName := f.BaseName + "-b"
+			// The CreateNamespace helper uses the input name as a Name Generator, so the namespace itself
+			// will have a different name than what we are setting as the value of ns-name.
+			// This is fine as long as we don't try to match the label as nsB.Name in our policy.
+			nsB, err := f.CreateNamespace(nsBName, map[string]string{
+				"ns-name": nsBName,
+			})
+			Expect(err).NotTo(HaveOccurred())
+
+			// Create Server with Service in NS-B
+			By("Creating a webserver tied to a service.")
+			serverPod, service := createServerPodAndService(f, nsA, "server", []int{80})
+			defer cleanupServerPodAndService(f, serverPod, service)
+			framework.Logf("Waiting for server to come up.")
+			err = framework.WaitForPodRunningInNamespace(f.ClientSet, serverPod)
+			Expect(err).NotTo(HaveOccurred())
+
+			// Create Policy for that service that allows traffic only via namespace B
+			By("Creating a network policy for the server which allows traffic from namespace-b.")
```

Comment:
sorry, didn't notice this before, but a bunch of the comments / log messages / object names here seem to have been copied from the namespace-only test, and need to be updated to clarify that this is testing combined namespaceSelector+podSelector